src/site/content/en/animations/animations-overview/index.md

```diff
@@ -58,22 +63,23 @@ changes further down the tree can sometimes result in layout calculations all th
 The larger the tree of visible elements,
 the longer it takes to perform layout calculations.
 
-## Animating paint properties
+### Animating paint properties {: #paint }
 
-If layout has been triggered then the next step is to move onto paint.
```

Comment:
I'm not sure if we need to repeat this because we explained it in the rendering pipeline section.
Maybe we do because the sequential nature of the rendering pipeline is the key thing to understand?